composer.json

```diff
@@ -29,9 +29,12 @@
     }
   },
   "require-dev": {
+    "ext-json": "*",
     "phpunit/phpunit": "^9.5",
```

Comment:
Now that `symfony/phpunit-bridge` is used and `./vendor/bin/simple-phpunit` is available, is it relevant to keep phpunit as a dev dep?

Reply:
We can actually delete phpunit/phpunit dependency ;) 
I'll take care of it

Reply:
Will we be able to keep using Pest with the simple-phpunit ? What about the watcher ?

Reply:
Normally, You shouldn't install this kind of dependency (like the watcher or phpunit), because it's specific to each developper

Reply:
@yoanbernabeu I am confused about the use of pestphp and phpunit-watcher, I don't think they are essential to test you bundle. WHat do you think ?

Reply:
> @yoanbernabeu I am confused about the use of pestphp and phpunit-watcher, I don't think they are essential to test you bundle. WHat do you think ?

Yes, let's keep it simple?

Reply:
So I can delete them from `composer.json` ?

Reply:
OK for me

Reply:
Comitted, you can resolve this conversation if you want ;)